src/proxy-animation.js

```diff
@@ -2,8 +2,10 @@ import {
   ScrollTimeline,
   addAnimation,
   removeAnimation,
-  relativePosition
+  relativeViewTimelinePosition,
```

Comment:
How about making relativePosition a member of ScrollTimeline with an override in ViewTimeline.  This approach would provide better encapsulation. The relativePosition method was added before non-view ScrollTimelines could have an animation range.  Then we can simply call this.timeline.relativePosition(offset) rather than handling view and scroll timelines separately.

Reply:
.done